books/free-programming-books-bn.md

```diff
@@ -26,6 +26,7 @@
 * [ReactJS](#reactjs)
 * [Svelte](#svelte)
 * [TypeScript](#typescript)
+* [Golang](#golang)
```

Comment:
Its not Golang, simply Go

Golang is used because the word go is so generic and common, however someone looking for go, will search for go here.

Reply:
"Go" is used in our other lists. In any case, the list must be alphabetized

Reply:
Resolved at https://github.com/davorpa/free-programming-books/commit/145277376ad8ad32d82668a2c7f1ebd202dfe263